### PR TITLE
ddl: update the behavior when 'RunWorker's' is false

### PR DESCRIPTION
**Pull request by zimulala**
If 'RunWorker' is false, we shouldn't become a owner too.

### CODE REVIEW

File: ddl/ddl_worker_test.go [hunk 1]
```diff
@@ -52,6 +52,53 @@ func (s *testDDLSuite) TestCheckOwner(c *C) {
 	c.Assert(d1.GetLease(), Equals, 2*time.Second)
 }
 
+func (s *testDDLSuite) TestRunWorker(c *C) {
```
Comment (shenli) [inline ddl/ddl_worker_test.go]: Add comments for the purpose of this test case.

**Comment by zimulala** [comment]
PTAL @shenli @coocood @winkyao

File: ddl/ddl_worker_test.go [hunk 2]
```diff
@@ -52,6 +52,54 @@ func (s *testDDLSuite) TestCheckOwner(c *C) {
 	c.Assert(d1.GetLease(), Equals, 2*time.Second)
 }
 
+// TestRunWorker tests whether the RunWorker workers when the value of RunWorker is false.
```
Comment (shenli) [inline ddl/ddl_worker_test.go]: workers -> works

Comment (coocood) [inline ddl/ddl_worker_test.go]: tests no job is done when the value of RunWorker is false.

**Comment by zimulala** [comment]
PTAL @coocood

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@zimulala We should cherry pick this to the release-1.0 .

**Pull request merged**